client/my-sites/plugins-wpcom/premium-plugins-panel.jsx

```diff
@@ -26,9 +30,10 @@ export const PremiumPluginsPanel = React.createClass( {
 
 				<Card className={ cardClasses }>
 					<div className="wpcom-plugins__list">
-						{ plugins.map( ( { name, descriptionLink, icon, plan, description } ) =>
-							<PremiumPlugin
-								{ ...{ name, key: name, descriptionLink, icon, plan, description } }
+						{ plugins.map( ( { name, descriptionLink, icon, category, description } ) =>
+							<Plugin
+								onClick={ () => onClick( name ) }
```

Comment:
This was difficult. The plugin component is the same regardless of the plan level, so the analytics code needs to know whether it's for a standard, premium, or business plugin. I tried hard to eliminate the fat-arrow function but didn't find any good way to do so. Granted, I _could_ eliminate it _per se_, but the underlying process (creating a new closure) was going to happen regardless. I chose to keep it here where it was obvious to someone reading the code.

For example, I could have made `onClick = name => () => dispatch( tracking( name ) )` but that would hide what's happening and I think that takes a step away from what we want semantically to happen.

Further, I didn't want to arbitrarily add props to the plugin piece itself or its `onClick` function because I felt that the _meaning_ of `onClick` should be just that: a generic function that gets called when the plugin is clicked, verses some specialized function taking specific parameters.

Feel free to disagree and point out why you think this is wrong.